app/controllers/v0/pension_claims_controller.rb

```diff
@@ -81,5 +71,37 @@ def log_validation_error_to_metadata(in_progress_form, claim)
       metadata['submission']['error_message'] = claim&.errors&.errors&.to_s
       in_progress_form.update(metadata:)
     end
+
+    def format_show_response(claim, state, form_submission, submission_attempt)
```

Comment:
These methods don't seem like they should belong in the control. To keep things clean, could you move them to a helper/module?